tests/ImageSharp.Tests/Formats/Bmp/BmpDecoderTests.cs

```diff
@@ -60,6 +60,158 @@ public void BmpDecoder_CanDecodeBitfields<TPixel>(TestImageProvider<TPixel> prov
             }
         }
 
+        [Theory]
+        [WithFile(Bit16Inverted, PixelTypes.Rgba32)]
+        [WithFile(Bit8Inverted, PixelTypes.Rgba32)]
+        public void BmpDecoder_CanDecode_Inverted<TPixel>(TestImageProvider<TPixel> provider)
+            where TPixel : struct, IPixel<TPixel>
+        {
+            using (Image<TPixel> image = provider.GetImage(new BmpDecoder()))
+            {
+                image.DebugSave(provider);
+                image.CompareToOriginal(provider);
+            }
+        }
+
+        [Theory]
+        [WithFile(Bit1, PixelTypes.Rgba32)]
+        [WithFile(Bit1Pal1, PixelTypes.Rgba32)]
+        public void BmpDecoder_CanDecode_1Bit<TPixel>(TestImageProvider<TPixel> provider)
+            where TPixel : struct, IPixel<TPixel>
+        {
+            using (Image<TPixel> image = provider.GetImage(new BmpDecoder()))
+            {
+                image.DebugSave(provider);
+                image.CompareToOriginal(provider, new SystemDrawingReferenceDecoder());
```

Comment:
I wonder if we understand on what images does ImageMagick fail? Any rule here?

Reply:
yes, it fails with 4-bit images

Reply:
just to clarify: i decided to execute the RLE tests for 8 bit only on windows, because System.drawing choses a different approach than ImageMagick to deal with the skipped pixels (System.Drawing == Black, Magick == First color of palette). The 8 bit tests were failing on linux.

Reply:
Let's put some comments in the source to explain that then please.

Reply:
i have added comments for the 4 bit tests. I have added an additional test for RLE 8 Bit with magick reference decoder